ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -264,6 +264,10 @@ def __int__(self):
     def __long__(self):
         return int(self._ivy_array)
 
+    @with_supported_dtypes({"2.6.0 and below": ("int32", "int64")}, "paddle")
+    def __mod__(self, y, /, name=None):
```

Comment:
A Question for the reviewer: a doubt with the implementation of mod method (within this same file)

which only takes one argument and hence the implementation of this method is same as well, however,

```python
            >>> import paddle

            >>> x = paddle.to_tensor([2, 3, 8, 7])
            >>> y = paddle.to_tensor([1, 5, 3, 3])
            >>> z = paddle.remainder(x, y)
            >>> print(z)
            Tensor(shape=[4], dtype=int64, place=Place(cpu), stop_gradient=True,
            [0, 3, 2, 1])

            >>> z = paddle.floor_mod(x, y)
            >>> print(z)
            Tensor(shape=[4], dtype=int64, place=Place(cpu), stop_gradient=True,
            [0, 3, 2, 1])

            >>> z = paddle.mod(x, y)
            >>> print(z)
            Tensor(shape=[4], dtype=int64, place=Place(cpu), stop_gradient=True,
            [0, 3, 2, 1])
```
the code under https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/tensor/math.py

seems to suggest something else which would mean that either our implementation is incorrect or I am looking at a wrong file

Reply:
Hi @druvdub,
I couldn't understand your doubt, could you please clerify it.

Reply:
Hi @NiteshK84, sorry I was just trying to understand if the implementation was right, mainly because in paddlepaddle all methods `remainder`, `floor_mod` and `mod` are supposed to have the same behaviour but we have defined it differently for all 3 here.

It should be fine if this implementation is fine. It is more of a personal doubt